cookbooks/helloworld/.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+inherit_from: .rubocop_todo.yml
+
```

Comment:
Perhaps we can add a comment here, eg:

#
# Default max of 80 is too short
#